testing/deployer/sprawl/internal/tfgen/templates/container-proxy.tf.tmpl

```diff
@@ -1,6 +1,6 @@
 resource "docker_container" "{{.DockerNetworkName}}-forwardproxy" {
   name     = "{{.DockerNetworkName}}-forwardproxy"
-  image = docker_image.nginx.latest
```

Comment:
Apparently this field was deprecated and i figured I'd update it with the new name while i was in here.